packages/host/app/resources/file.ts

```diff
@@ -143,14 +147,19 @@ class _FileResource extends Resource<Args> {
 
     let content = await response.text();
     let self = this;
+    // Inside test, The loader occasionally doesn't do a network request and creates Response object manually
+    // This means that reading response.url will give url = '' and we cannot manually alter the url in Response
+    // The below condition is a workaround
+    // TODO: CS-5982
+    let url = response.url == '' ? this._url : response.url;
```

Comment:
Until this is researched/fixed, would be more appropriate to do this only in test environment? I.e. `if (environment === 'test') { ... }`